app/views/shared/_venue.html.haml

```diff
@@ -1,19 +1,21 @@
+- formatted_address = address.to_html
+- section_title = formatted_address.empty? ? "Host" : "Venue"
```

Comment:
Dirty way to change  the section title to host if the venue has no address. Should remove this once we add proper virtual support to Events.